keras/engine/training.py

```diff
@@ -1332,6 +1409,14 @@ class indices (integers) to
                 sample_weight_mode="temporal" in compile().
             initial_epoch: epoch at which to start training
                 (useful for resuming a previous training run)
+            steps_per_epoch: Total number of steps (batches of samples)
+                before declaring one epoch finished and starting the
+                next epoch. The default `None` is equal to the number
+                of unique samples in your dataset divided by the batch
+                size, or 1 if that cannot be determined.
```

Comment:
Add a node about how this is only relevant if training from TensorFlow data tensors.